python/docs/source/usage.rst

```diff
@@ -484,6 +484,53 @@ the data passed to it differs from the existing table's schema. If you wish to
 alter the schema as part of an overwrite pass in ``overwrite_schema=True``.
 
 
+Updating Delta Tables
+---------------------
+
+.. py:currentmodule:: deltalake.table
+
+Row values in an existing delta table can be updated with the :meth:`DeltaTable.update` command. A update
+dictionary has to be passed, where they key is the column you wish to update, and the value is a
+Expression in string format.
+
+Update all the rows for the column "deleted" to the value True.
```

Comment:
TBH, I don't love this example, since the values might confuse a user who doesn't know the different between this and row deletion. Could you change the column name `deleted` to something else? Perhaps `processed`?